### PR TITLE
Update readme, alias commands and check list to follow that of TGP-2302

### PR DESCRIPTION
**Pull request by Reece-Carruthers**
# Checklist for developers before reviewing
 - [x] I have merged in main
 - [x] I have run  `sbt clean coverage test it/test coverageReport`
 - [ ] I have written unit tests to cover any code that I wrote
 - [x] I have run `sbt scalafmt` and `sbt test:scalafmt`
 - [x] I have updated the microservices
 - [ ] I have checked all the Acceptance Criteria pass
 - [ ] I have run the pipeline journey tests
 - [ ] I have run the local journey tests
 - [ ] I have moved the ticket to Review in Jira
 - [x] I have left a message in tgp-prs on slack to alert other Devs that this ticket is ready for review

# Checklist for developers before merging
 - [x] The PR has been approved by 2 other developers
 - [x] I have merged in main
 - [x] I have run  `sbt clean coverage test it/test coverageReport`
 - [x] I have run `sbt scalafmtCheckAll`
 - [x] I have updated the microservices
 - [ ] I have checked all the Acceptance Criteria pass
 - [ ] I have run the pipeline journey tests
 - [ ] I have run the local journey tests
 - [x] I have spoken to the QAs about any changes that may need to be made to the journey tests OR no changes to the journey tests are necessary

# Reminders for developers after merging
 1. Wait for pipeline to pass
 2. Move ticket to QA column in Jira
 3. Leave message in tgp-qa on slack to alert QAs that this ticket is ready for testing

### CODE REVIEW

File: pull_request_template.md [hunk 1]
```diff
@@ -1,27 +1,25 @@
...
+- [ ] I have checked all the Acceptance Criteria pass
+- [ ] I have run the pipeline journey tests
+- [ ] I have run the local journey tests
+- [ ] I have spoken to the QAs about any changes that may need to be made to the journey tests OR no changes to the journey tests are necessary
 
 # Reminders for developers after merging
- 1. Wait for pipeline to pass
- 2. Move ticket to QA column in Jira
- 3. Leave message in tgp-qa on slack to alert QAs that this ticket is ready for testing
+1. Wait for pipeline to pass
```
Comment (besscerule) [inline pull_request_template.md:23]: You need to start the pipeline in this case not wait because it only auto triggers when the frontend is merged in

Comment (besscerule) [reply pull_request_template.md:23]: ignore this comment - i don't know what I'm talking about

Comment (Reece-Carruthers) [reply pull_request_template.md:23]: No problem :laughing:

File: README.md [hunk 2]
```diff
@@ -87,6 +87,38 @@ Run this to format non-test scala files: `sbt scalafmt`
...
 
+### SBT Alias Commands
+
+To run all unit and IT tests, check code coverage, and generate a report: Use `sbt testAndCoverage`
+
+Before raising a PR, to run all unit and IT tests, format the code, check code coverage, and generate a report: Use `sbt prePR`
+
+Before merging, to run all unit and IT tests, verify code formatting, check code coverage, and generate a report: Use `sbt preMerge`
+
+### Testing
```
Comment (besscerule) [inline README.md:98]: I think you have some duplication in this file - please double check and where possible make it match the layout and order of the frontend read me

Comment (Reece-Carruthers) [reply README.md:98]: :facepalm: not sure how I've managed that

**Review by besscerule — APPROVED** [review]

**Review by yohan-oc — APPROVED** [review]

**Pull request merged**